packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/init.js

```diff
@@ -0,0 +1,13 @@
+import { transformPermissionsData } from './utils';
+
+const init = (state, permissions = []) => {
```

Comment:
we don't need a lot of the stuff here, we don't have this level of granularity on permissions

Reply:
you are right, we have removed all the init file